### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by awojtczyk**

### CODE REVIEW

**Comment by woudsma** [comment]
Closes #89

File: src/ansible/inventory/inventory.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 all:
   vars:
     ansible_python_interpreter: /usr/bin/python3
-    remote_user: root
-    ansible_ssh_user: root
+    remote_user: ubuntu
```
Comment (n-rodriguez) [inline src/ansible/inventory/inventory.yml:4]: What if I use RedHat?

Comment (n-rodriguez) [reply src/ansible/inventory/inventory.yml:4]: Maybe a more generic user would be preferable? wdyt?

Comment (n-rodriguez) [reply src/ansible/inventory/inventory.yml:4]: Or maybe we should focus on a small set of supported distros ? (Ubuntu and Debian at first)

Or both? (generic user + small set of distros)

Comment (awojtczyk) [reply src/ansible/inventory/inventory.yml:4]: > Maybe a more generic user would be preferable? wdyt?

you are right, that was intended to fix the aws issues, before the Ubuntu idea :)

Comment (n-rodriguez) [reply src/ansible/inventory/inventory.yml:4]: > you are right, that was intended to fix the aws issues, before the Debian idea :)

btw I'm doing some tests on a single node and except some already known bugs and some missing parts in the doc it works very well :) :+1:  :heart:

**Comment by awojtczyk** [comment]
Closes #114 
Closes #112

**Pull request merged**